### DISCUSSION TITLE
Suppress class reuse

### DISCUSSION THREAD

**Discussion by mwhellwig** [nlohmann/json#4360] [General]
I generated C++ code from a JSON schema. The schema has a field called acceleration which contains sub-fields: east, north, up and accuracy. The accuracy field also contains east, north and up (but no accuracy). The generator reused the quicktype::acceleration class for accuracy. The problem with this is that the accuracy field now has an accuracy sub-field. This shows up in the JSON output. Is there a way to suppress the reuse of classes?

**Comment by nlohmann** [comment]
Can you provide an example?

**Comment by gregmarr** [comment]
@mwhellwig Did you mean to ask this in https://github.com/glideapps/quicktype ?
